healthcheck_test.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestEndpoints(t *testing.T) {
+	t.Run("should return 200", func(t *testing.T) {
```

Comment:
nitpick: the naming of all this test stuff could be improved. This is what it looks like when I run it in goland:
![image](https://github.com/user-attachments/assets/791bc7ce-aa63-4fa8-82d0-733774ae9537)

So:
 - The file is named "healthcheck_test.go" but you also have the 404 test in here
 - The function name of "TestEndpoints" is package-wide not file-specific, so unless you intend to put all your endpoint tests in this function, come up with something more specific
 - For "should return 200", this name needs to have something about the health endpoint in it, and maybe not be so specific to the assert
 - [insert similar comment about "should return 404"]

Naming is [hard](https://www.reddit.com/r/ProgrammerHumor/comments/cmopap/hardest_thing_in_programming/) lol